metrix-mapping/src/main/java/com/powsybl/metrix/mapping/MappableEquipmentType.java

```diff
@@ -18,8 +18,13 @@
     HVDC_LINE("hvdcLine"),
     LOAD("load"),
     BOUNDARY_LINE("boundaryLine"),
-    PST("pst"),
-    SWITCH("breaker");
+    SWITCH("breaker"),
+    TRANSFORMER("transformer"),
+    PHASE_TAP_CHANGER("transformer"),
+    RATIO_TAP_CHANGER("transformer"),
+    LCC_CONVERTER_STATION("lccConverterStation"),
+    VSC_CONVERTER_STATION("vscConverterStation"),
+    PST("pst"); // Kept for compatibility
```

Comment:
Maybe it should be deprecated to flag in the code this should not be used anymore (except for backward compatibility)